src/test/versions_test.ts

```diff
@@ -29,143 +29,204 @@ const defaultBrowser = {
 };
 
 suite('versions', () => {
-  test('makeServerPlans', async () => {
-    const specs: BenchmarkSpec[] = [
-      // mybench running with two custom versions.
-      {
-        name: 'mybench',
-        url: {
-          kind: 'local',
-          urlPath: '/mylib/mybench/',
-          version: {
-            label: 'v1',
-            dependencyOverrides: {
-              mylib: '1.0.0',
+  suite('makeServerPlans', async () => {
```

Comment:
@bicknellr Recommend viewing with ?w=1 because the test didn't change as much as it looks.